RussianFilter/sections/specific.txt

```diff
@@ -32,6 +32,7 @@ news.allcrimea.net###kinotvukrnet
 tut.by##div[data-io-title="Новости компаний"]
 masterpaiki.ru###secondary .widget_execphp
 masterpaiki.ru##a[href^="http://epnclick.ru/redirect"]
+filmitorrent.site##.post > center > a[target="_blank"] > img
```

Comment:
It seems that this rule doesn't block/hide second ad (please look at the second screenshot here - https://github.com/AdguardTeam/AdguardFilters/issues/38085), so maybe `center > a[target="_blank"] > img` will be okay?

Reply:
@AdamWr,
changed the rule